### ISSUE TITLE
How to improve download speed?

### ISSUE THREAD

**Issue by Dzna10** [gram-js/gramjs#647]
Is there any way to improve download speed? Currently if I don't download multiple files at once the download speed sucks a lot.

**Comment by alvvaysxxx** [comment]
I suppose the only way to increase it is to buy Telegram Premium

**Comment by Dzna10** [comment]
Yeah, I think so too. Anyway, thanks for your comment!

**Comment by liesauer** [comment]
the only way? if i use the official app with proxy, it can reach 1MB/s, but gramjs with the same proxy, i get 100KB/s

**Comment by DanilochTop** [comment]
Yep, you are right. That's a great point. I don't know what is happening with the code to cause this slowdown, maybe they are using some method that Telegram doesn't use anymore. If anyone does fix this, make sure to post in this thread the github repo, please!

**Comment by SaeedDev94** [comment]
I have telegram premium but the max speed I got is 10 Mib/s which is too slow !!  
My machine can reach 1024 Mbit/s easily but I don't know why the download speed is very slow :(
